tsdb/index/index.go

```diff
@@ -1443,18 +1443,23 @@ func (r *Reader) SymbolTableSize() uint64 {
 // SortedLabelValues returns value tuples that exist for the given label name.
 // It is not safe to use the return value beyond the lifetime of the byte slice
 // passed into the Reader.
-func (r *Reader) SortedLabelValues(name string) ([]string, error) {
-	values, err := r.LabelValues(name)
+func (r *Reader) SortedLabelValues(name string, matchers ...*labels.Matcher) ([]string, error) {
+	values, err := r.LabelValues(name, matchers...)
 	if err == nil && r.version == FormatV1 {
 		sort.Strings(values)
 	}
 	return values, err
 }
 
 // LabelValues returns value tuples that exist for the given label name.
+// Passing matchers is not supported yet.
```

Comment:
```suggestion
// Passing matchers is not supported yet.
```

should that be a TODO: ? otherwise, remove `yet`

Reply:
yes, i'll make it a TODO